code/modules/projectiles/projectile/reusable.dm

```diff
@@ -38,6 +34,24 @@
 	icon_state = "foamdart"
 	ammo_type = /obj/item/ammo_casing/caseless/foam_dart
 	range = 10
+	var/obj/item/weapon/pen/pen = null
```

Comment:
the references must be deleted in /obj/item/projectile/bullet/reusable/foam_dart/Destroy()